Directory.Build.props

```diff
@@ -56,4 +56,10 @@
     <Compile Include="$(MSBuildThisFileDirectory)GlobalUsings.cs" />
   </ItemGroup>
 
+  <!-- Set the version and local path for Sentry CLI (downloaded in the restore phase of Sentry.csproj) -->
+  <PropertyGroup Condition="'$(SolutionName)' != 'Sentry.Unity'">
+    <SentryCLIVersion>2.11.0</SentryCLIVersion>
```

Comment:
One additional thing we could add here for security reason is to take besides the version, the checksum to verify what we downloaded is what we want.

Doesn't seem like we're doing this on the fastlane plugin anymore: https://github.com/getsentry/sentry-fastlane-plugin/blob/9e694cb721ccb0f4e084a54da305226cabd94de3/script/sentry-cli-download.sh#L15 or grade.

@vaind did the sentry-cli PR bump automation, maybe he knows about this

Reply:
I saw the checksum verification in the sentry-cli download bash script.  It uses python.  I think I can do something similar with C# or pure MSBuild so we don't have a python dependency.

Reply:
Was able to do this easily with the [`VerifyFileHash`](https://learn.microsoft.com/visualstudio/msbuild/verifyfilehash-task) msbuild task.